tune/tune.c

```diff
@@ -148,7 +148,7 @@ int main(int argc, char *argv[])
 		switch (c) {
 		case 'l':
 			if (optarg) {
-				snprintf(label_input, 11, "%s", optarg);
+				snprintf(label_input, VOLUME_LABEL_MAX_LEN, "%s", optarg);
```

Comment:
`snprintf(buffer, sizeof(buffer), ...)` is correct usage of `snprintf` for static allocated buffer.